From 0d22be5a4f5fb31a45fb4c55dde5037bfba3c572 Mon Sep 17 00:00:00 2001
From: Gu Zitao <guzitao@wxiat.com>
Date: Tue, 28 Jun 2022 08:51:26 +0800
Subject: [PATCH] sw64: rename CONFIG_HAVE_GENERIC_GUP to CONFIG_HAVE_FAST_GUP
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 0d22be5a4f5fb31a45fb4c55dde5037bfba3c572
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I5GFRP

--------------------------------

This patch is a backport of commit 67a929e097b7 ("mm: rename
CONFIG_HAVE_GENERIC_GUP to CONFIG_HAVE_FAST_GUP"). In order to
use the generic GUP, let's do the same.

Signed-off-by: Gu Zitao <guzitao@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/sw_64/Kconfig b/arch/sw_64/Kconfig
index ec6e583a5d9a..cf2f6f00708c 100644
--- a/arch/sw_64/Kconfig
+++ b/arch/sw_64/Kconfig
@@ -7,7 +7,7 @@ config SW64
 	select HAVE_OPROFILE
 	select HAVE_PCSPKR_PLATFORM
 	select HAVE_PERF_EVENTS
-	select HAVE_GENERIC_GUP
+	select HAVE_FAST_GUP
 	select GENERIC_CLOCKEVENTS
 	select GENERIC_IRQ_PROBE
 	select GENERIC_IRQ_LEGACY
-- 
2.33.0

